### PR TITLE
Feature/machine account

### PR DESCRIPTION
**Pull request by dwlfrth**
## Description

Keep track of machine auth EAP connections by keeping the hostname in 'node'
## Impacts

Shouldn't have any.
## NEWS file entries

New Features
++++++++++++

Enhancements
++++++++++++
- We keep track of EAP machine auth by keeping the hostname in the 'node' table.

Bug Fixes
+++++++++
## Delete branch after merge

YES

### CODE REVIEW

**Comment by dwlfrth** [comment]
Please review

File: lib/pf/radius.pm [hunk 1]
```diff
@@ -122,6 +122,12 @@ sub authorize {
     # There is activity from that mac, call node wakeup
     node_mac_wakeup($mac);
 
+    # Handling machine auth detection
+    if ( defined($user_name) && $user_name =~ /host\// ) {
```
Comment (fdurand) [inline lib/pf/radius.pm]: What happen if the username is ghost/robert ?

Comment (julsemaan) [inline lib/pf/radius.pm]: Or if the username is zammithost/ludovic ?

Comment (extrafu) [inline lib/pf/radius.pm]: A sAMAccountName cannot contain a forward slash.

http://social.technet.microsoft.com/wiki/contents/articles/11216.active-directory-requirements-for-creating-objects.aspx#Objects_with_sAMAccountName_Attribute

Comment (fdurand) [inline lib/pf/radius.pm]: But a domain/sAMAccountName yes

Comment (extrafu) [inline lib/pf/radius.pm]: Isn't it normally domain\sAMAccountName ?

https://msdn.microsoft.com/en-us/library/windows/desktop/aa380525%28v=vs.85%29.aspx#down_level_logon_name

**Comment by dwlfrth** [comment]
Based on Phil Mayers (FreeRADIUS): http://lists.freeradius.org/pipermail/freeradius-users/2012-December/064229.html

**Comment by dwlfrth** [comment]
Soo ?

**Comment by fdurand** [comment]
If we do TLS and the username inside is ghost/robert ?
Can we just have a regex like /^host\// ?

File: lib/pf/radius.pm (hunk 1 above)
Comment (louismunro) [inline lib/pf/radius.pm]: why not just /^host\//?

**Comment by dwlfrth** [comment]
Let me fix that fo u ;)

**Comment by dwlfrth** [comment]
Me want to be merged

**Comment by fdurand** [comment]
Will be merged after 4.6.1

**Comment by fdurand** [comment]
can you rebase ?

**Comment by dwlfrth** [comment]
Done

**Pull request merged**